### ISSUE TITLE
Crash while using an editor

### ISSUE THREAD

**Issue by Delnar** [4ian/GDevelop#1027]
=> Please write here a short description of when the error occured and how to reproduce it.
You also may have to create an account on GitHub before posting.

When you're ready, click on "Submit new issue". Don't change the rest of the message. Thanks!

## Error stack (don't write anything here)
```
TypeError: Cannot read property 'existsSync' of null
    at l (https://editor.gdevelop-app.com/static/js/1.1e8a4f32.chunk.js:1:551087)
    at https://editor.gdevelop-app.com/static/js/1.1e8a4f32.chunk.js:1:228975
    at Array.forEach (<anonymous>)
    at t.n.checkMissingPaths (https://editor.gdevelop-app.com/static/js/1.1e8a4f32.chunk.js:1:228938)
    at t.value (https://editor.gdevelop-app.com/static/js/1.1e8a4f32.chunk.js:1:229360)
    at pa (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:354611)
    at ha (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:350226)
    at la (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:349842)
    at va (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:356083)
    at kn (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:301170)
    at HTMLDocument.i (https://editor.gdevelop-app.com/static/js/53.f92ae050.chunk.js:1:820910)
```

## Component stack (don't write anything here)
```

    in t
    in div
    in div
    in t
    in div
    in t
    in Unknown
    in t
    in t
    in div
    in t
    in t
    in t
    in Unknown
    in div
    in div
    in t
    in div
    in div
    in t
    in t
    in t
    in div
    in t
    in t
    in t
    in t
    in t
    in t
    in t
    in n
    in t
    in t
    in t
    in DragDropContext(t)
    in t
    in t
```

**Comment by 4ian** [comment]
Thanks for the report, fixed for next version in 1de33312f31a22d8c2e298bc7759ee504a7745a3

@blurymind There was a crash due to path checks, remember to always test on the web-app ;) When using a module that is imported using `optionalRequire` (like fs), it can be null and code should check for this before using it (otherwise, boom 💥😄)

**Comment by blurymind** [comment]
Ah this one slipped by me somehow. Sorry about that :)
